src/app/tests/suites/certification/Test_TC_WNCV_3_5.yaml

```diff
@@ -77,48 +77,44 @@ tests:
     - label: "2c: TH reads OperationalStatus attribute from DUT"
       command: "readAttribute"
       attribute: "OperationalStatus"
-      PICS: WNCV.S.A000a
+      PICS: WNCV.S.A000a && ( WNCV.S.F00 || WNCV.S.F01 )
       response:
           value: 0
 
     ######## DownOrClose Long-Run: Category only for Position Aware DUT #######
     ### Step 3x -> Verify the DUT has reached its fully-open limits
     ### Mandatory Depends on the FeatureMap
-    - label:
-          "3a: If (PA & LF) TH reads CurrentPositionLiftPercent100ths attribute
-          from DUT"
+    - label: "3a: TH reads CurrentPositionLiftPercent100ths attribute from DUT"
       command: "readAttribute"
       attribute: "CurrentPositionLiftPercent100ths"
-      PICS: WNCV.S.F00 && WNCV.S.F02 && WNCV.S.A000e
+      PICS: WNCV.S.A000e && ( WNCV.S.F00 || WNCV.S.F01 )
```

Comment:
attribute CurrentPositionLiftPercent100ths is mandatory if LF & PA_LF -> WNCV.S.F00 &&  WNCV.S.F02

Reply:
Removed changes from PR please verify it.